changelogs/unreleased/fix-escaping-issue-in-install-docs.yml

```diff
@@ -0,0 +1,5 @@
+description: Fix dollar sign escaping issue in installation documentation
+change-type: patch
+destination-branches: [master]
```

Comment:
```suggestion
destination-branches: [master, iso4]
```
Or am I missing something?

Reply:
This is the installation documentation for inmanta OSS, so `master` should be sufficient.

Reply:
I see it's present on the iso4 branch as well but I assume it gets replaced with the iso instructions then?

Reply:
Yes, it is.